cmd/esc/cli/testdata/env-tag-update-key.yaml

```diff
@@ -0,0 +1,17 @@
+run: |
+  esc env tag test-org/env owner pulumi
+  esc env tag test-org/env owner team pulumi
```

Comment:
ergonomics feel a little weird here. would prefer a `tag mv` command like

```console
> esc env tag mv test-org/env owner team
```

Reply:
yeah, I was trying to stick with the same pattern we employ in `esc env tag org/env@tag` since that will both create or move a tag but I can make this explicit